ansible/roles/etckeeper/defaults/main.yml

```diff
@@ -176,6 +176,9 @@ etckeeper__default_gitignore:
   - name: 'zfs-zpool-cache'
     ignore: 'zfs/zpool.cache'
 
+  - name: 'etesync-secret'
+    ignore: 'etesync-server/secret.txt'
```

Comment:
That's a bit random, isn't it? It's the first time I hear about [EteSync](https://www.etesync.com), it's not even available in Debian. I don't see an RFP or ITP messages about it either. There's no efforts to create a `.deb` package in its GitHub repositories that I could see on a quick glance, looking for an Ansible playbook/role for it also doesn't show any results, nothing on Ansible Galaxy...

The `etckeeper` role is designed to handle such cases via Ansible inventory perfectly fine, so why add this particular case to the role defaults?

Reply:
I guess because I have a DebOps role 80 % done for a EteSync server and adding it to etckeeper ignores appeared as one usual preparation work for me :)

Reply:
Very well then, it can stay. :-)